Makefile

```diff
@@ -52,6 +52,12 @@ docs: ## Build project documentation in live reload for editing
 # Spins up livereload environment for editing; blocks.
 	rm -rf _build/ && sphinx-autobuild . _build/html
 
+.PHONY: docs-lint
+docs-lint: ## Check documentation for common syntax errors.
+# The `-W` option converts warnings to errors.
+# The `-n` option enables "nit-picky" mode.
+	rm -rf _build/ && sphinx-build -Wn . _build/html
```

Comment:
We have a `clean` target, how about just `make clean &&` (can add the same for the main `docs` target while we're at it). That way we avoid repeating the build directory all over the place, in case we ever want to change it.